deps/openssl.cmake

```diff
@@ -22,13 +22,14 @@ else (WithSharedOpenSSL)
       set(OPENSSL_BUILD_COMMAND nmake)
   else()
       set(OPENSSL_CONFIGURE_COMMAND ./config ${OPENSSL_CONFIG_OPTIONS})
-      set(OPENSSL_BUILD_COMMAND make)
+      set(OPENSSL_BUILD_COMMAND make -j6)
```

Comment:
Why hard-code this to 6?

Reply:
I found out while working on this, the OPENSSL_BUILD_COMMAND doesn't pass down the parallel build job. We should remove the -j6 in favor of passing down the make options from the parent cmake.